xikolo-ios/ViewControllers/Courses/DropdownTableViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DropdownTableViewController.swift
+//  xikolo-ios
+//
+//  Created by Bjarne Sievers on 13.09.16.
+//  Copyright © 2016 HPI. All rights reserved.
+//
+
+import UIKit
+
+class DropdownTableViewController: UITableViewController {
```

Comment:
When a user is on the course item table view page, he can now click on the navigation bar and gets a dropdown containing a tableview, where he can choose between "Learnings", "Discussion" and "Course Details". This is the VC for this static tableview.